src/client/pages/OfferNew/Perils/PerilModal.tsx

```diff
@@ -1,253 +1,137 @@
 import styled from '@emotion/styled'
-import { colorsV3, fonts } from '@hedviginsurance/brand'
-import hexToRgba from 'hex-to-rgba'
-import React from 'react'
+import { colorsV3 } from '@hedviginsurance/brand'
+import React, { useState, useEffect } from 'react'
 import { BackArrow } from 'components/icons/BackArrow'
-import { Checkmark } from 'components/icons/Checkmark'
-import { Crossmark } from 'components/icons/Crossmark'
 import { ForwardArrow } from 'components/icons/ForwardArrow'
-import { InfoIcon } from 'components/icons/Info'
 import { Modal, ModalProps } from 'components/ModalNew'
 import { PerilV2 } from 'data/graphql'
 import { useTextKeys } from 'utils/textKeys'
+import {
+  MEDIUM_SMALL_SCREEN_MEDIA_QUERY,
+  LARGE_SCREEN_MEDIA_QUERY,
+  MEDIUM_SCREEN_MEDIA_QUERY,
+} from 'utils/mediaQueries'
+import { Headline } from 'components/Headline/Headline'
+import { SelectedOptionCheckmark } from 'components/icons/SelectedOptionCheckmark'
+import { Cross } from 'components/icons/Cross'
+import { InfoIconFilled } from 'components/icons/Info'
 
 const TRANSITION_MS = 250
 
+type Color = 'black' | 'gray'
```

Comment:
nit: could be useful to know what this color is for - is it for the entire modal or something else?

```suggestion
type ListItemColor = 'black' | 'gray'
```